packages/frontend/src/components/ProjectTablesConfiguration/ProjectTablesConfiguration.tsx

```diff
@@ -136,6 +137,45 @@ const ProjectTablesConfiguration: FC<{
         }
     }, [isSuccess, onSuccess]);
 
+    const setFormValuesFromData = useCallback(() => {
+        if (data) {
+            form.setValues({
+                type: data.tableSelection.type,
+                tags:
+                    data.tableSelection.type === TableSelectionType.WITH_TAGS &&
+                    data.tableSelection.value
+                        ? data.tableSelection.value
+                        : [],
+                names:
+                    data.tableSelection.type ===
+                        TableSelectionType.WITH_NAMES &&
+                    data.tableSelection.value
+                        ? data.tableSelection.value
+                        : [],
+            });
+
+            form.resetDirty({
+                type: data.tableSelection.type,
+                tags:
+                    data.tableSelection.type === TableSelectionType.WITH_TAGS &&
+                    data.tableSelection.value
+                        ? data.tableSelection.value
+                        : [],
+                names:
+                    data.tableSelection.type ===
+                        TableSelectionType.WITH_NAMES &&
+                    data.tableSelection.value
+                        ? data.tableSelection.value
+                        : [],
+            });
```

Comment:
this seems repetitive. can we have a one object and use that to call both of the functions? more questions about those two functions below

Reply:
@IrakliJani  Thanks for the review, will update this to remove the repetition.